packages/fake-data/src/fakeAssessmentSteps.ts

```diff
@@ -27,7 +30,15 @@ const generateAssessmentStep = (
       en: `${faker.lorem.word()} EN`,
       fr: `${faker.lorem.word()} FR`,
     },
-    poolSkills: [],
+    poolSkills: [
+      {
+        id: faker.string.uuid(),
+        type: faker.helpers.arrayElement<PoolSkillType>(
+          Object.values(PoolSkillType),
+        ),
+        skill: fakeSkills(1)[0],
+      },
+    ],
```

Comment:
Trying to see what broke Jest I think this may have a part. 
This changes the data seen in `/web/src/components/AssessmentStepTracker/testData.ts` which is used by the step tracker test. 

The assessment step tracker story is helpful because I think it uses the same data. 

Main

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/30e36ba9-b16c-4388-8d7f-28782004e5f9)

Branch

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/b9ffe074-b2d5-4ab6-95f3-a9aebc46a2ae)

Reply:
Thanks Vachan, I was looking for this! Fixed by [1e78cb4](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9324/commits/1e78cb40c0ff72d27e4651be160cc08f8c27049d)